packages/atauth/CMakeLists.txt

```diff
@@ -9,8 +9,12 @@ set(
   ${CMAKE_CURRENT_LIST_DIR}/src/send_enroll_request.c
   ${CMAKE_CURRENT_LIST_DIR}/src/atactivate_arg_parser.c
   ${CMAKE_CURRENT_LIST_DIR}/src/atactivate.c
+  ${CMAKE_CURRENT_LIST_DIR}/src/auth_cli.c
   ${CMAKE_CURRENT_LIST_DIR}/src/atauth_build_atkeys_file_path.c
   ${CMAKE_CURRENT_LIST_DIR}/src/atauth_fetch_home_dir.c
+        ../../examples/desktop/at_talk/src/main.c
+        ../../examples/desktop/connection_hooks/main.c
+        ../../examples/desktop/crud/delete.c
```

Comment:
These source files should not be here

Reply:
This has been addressed in the latest commit '[80f123c](https://github.com/atsign-foundation/at_c/pull/453/commits/80f123cf7f73d1e85060a92c8240e61da5f01086)'